### ISSUE TITLE
Announce battery % when rider gets on board

### ISSUE THREAD

**Issue by drewbaumann** [OnlyInAmerica/OneWheelMonitor#1]
When you get on the board it would be nice to hear the current percentage

**Comment by OnlyInAmerica** [comment]
@drewbaumann definitely! The app currently announces battery on board connection (if earbuds are attached) but I don't yet have a mechanism to re-deliver that alert if earbuds are plugged in shortly after. Will figure this out for next release.

**Comment by drewbaumann** [comment]
Here is the user story: I'm cruising and have not really been thinking about percentage. I hear it go off during the regular intervals, but forget if I am currently at 40 or 30 percent. I stop and get off the board to get my feet feeling a little better. Then I get back on the board and hear my current percentage. Think of it like a helpful welcome/dossier. It could be as simple as "Rider on. Battery at 34 percent." Or perhaps "Rider on. Battery at 34 percent. You have 2.1 miles remaining."

**Comment by OnlyInAmerica** [comment]
Hey sorry I missed your last comment but I'm testing the "Rider on" report now. Thanks for clarifying.

The board doesn't report mileage directly so that figure will be a rough estimate based on trip odometer and remaining battery, but should be somewhat useful!

**Comment by OnlyInAmerica** [comment]
Just an update - A basic "Rider on" report was added in 1.3, but for now it just announces battery percentage. Still working on ride remaining distance estimator.
